CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ project(
   VERSION 0.1.0)
 set(CMAKE_C_STANDARD 99)
 set(CMAKE_CXX_STANDARD 17)
+add_compile_definitions(WEBP_HAVE_PNG)
```

Comment:
This line is not needed anymore: please rebase your branch to get the image library fixes.